js/views/userPage/CategoryFilter.js

```diff
@@ -23,7 +23,7 @@ export default class extends BaseVw {
   }
 
   className() {
-    return 'clrP clrBr padMd clrT contentBox clrSh2 categoryFilter';
+    return 'clrP clrBr padMd clrT contentBox clrSh2 form veryCompact categoryFilter';
```

Comment:
Adding the form and veryCompact classes allows the store category radio buttons to use the normal radio button styling.